SDK/AppCenterPush/Microsoft.AppCenter.Push.UWP/Microsoft.AppCenter.Push.UWP.csproj

```diff
@@ -11,12 +11,13 @@
     <AssemblyName>Microsoft.AppCenter.Push</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.14393.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
```

Comment:
Please revert this file, this is not part of the version bump script and should not be touched during a release.